index.html

```diff
@@ -43,7 +46,68 @@
 
     <div class="homepage-main">
         <div class="homepage-content">
-            <p>This is main</p>
+            <div class="genre-cards-container">
+                <div class="genre-cards"><a href="Pages/Product-Page/product.html"><h2>Fiction</h2></a></div>
+                <div class="genre-cards"><a href="Pages/Product-Page/product.html"><h2>Sci-fi</h2></a></div>
```

Comment:
just a suggestion could reduce the size of the font and the genre-cards

Reply:
Done in next PR. 👍